src/app/modules/customer-management/store/customer-management.actions.ts

```diff
@@ -8,6 +8,13 @@ export enum CustomerManagementActionTypes {
   LOAD_CUSTOMERS = '[CustomerManagement] LOAD_CUSTOMERS',
   LOAD_CUSTOMERS_SUCCESS = '[CustomerManagement] LOAD_CUSTOMERS_SUCCESS',
   LOAD_CUSTOMERS_FAIL = '[CustomerManagement] LOAD_CUSTOMERS_FAIL',
+  DELETE_CUSTOMER = '[CustomerManagement] DELETE_CUSTOMER',
+  DELETE_CUSTOMER_SUCCESS = '[CustomerManagement] DELETE_CUSTOMER_SUCCESS',
+  DELETE_CUSTOMER_FAIL = '[CustomerManagement] DELETE_CUSTOMER_FAIL',
+  UPDATE_CUSTOMER = '[CustomerManagement] UPDATE_CUSTOMER',
+  UPDATE_CUSTOMER_SUCCESS = '[CustomerManagement] UPDATE_CUSTOMER_SUCCESS',
+  UPDATE_CUSTOMER_FAIL = '[CustomerManagement] UPDATE_CUSTOMER_FAIL',
+  SET_CUSTOMER_ID_TO_EDIT = '[CustomerManagement] SET_CUSTOMER_ID_TO_EDIT',
```

Comment:
could implement the RESET_CUSTOMER_ID_TO_EDIT, it cleans the variable and allows that you can edit (select) the same item many times.

Reply:
done